internal/security/fileprovider/init.go

```diff
@@ -70,7 +70,7 @@ func (b *Bootstrap) BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ sta
 		lc.Info("bypassing certificate verification for secret store connection")
 		req = secretstoreclient.NewRequestor(lc).Insecure()
 	}
-	vaultScheme := cfg.SecretService.Scheme
+	vaultScheme := cfg.SecretService.Protocol
```

Comment:
For consistency can we change `vaultScheme` to `vaultProtocol`?

Reply:
yes, that makes sense